codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,23 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{num_questions_answered}} question{{ num_questions_answered|pluralize }} in the last month!</p>
+<!--        {% include "users/activity_graph.html" %}-->
```

Comment:
What's this?

Reply:
I think this is supposed to be similar to the github activity graph, don't think it quite works though.

Do we leave the `activity-graph.html` file in and remove the commented out reference or remove everything all together?

![image](https://user-images.githubusercontent.com/16066822/77878546-0662b280-72b5-11ea-9c4b-c4a5b8ac7d45.png)